frontend/package.json

```
@@ -24,7 +27,12 @@
     "@types/estree": "^1.0.9",
     "@types/randomcolor": "^0.5.9",
     "@types/zxcvbn": "^4.4.5",
+
+    "@vercel/analytics": "^1.3.0",
+    "@vercel/speed-insights": "^1.0.11",
```

Biome (2.5.1)

[error] The key `@vercel/speed-insights` was already declared.

(lint/suspicious/noDuplicateObjectKeys)